migrations/versions/20c10335b553_add_project_livestream_urls.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+"""add project livestream urls
+
+Revision ID: 20c10335b553
+Revises: c11dc931b903
+Create Date: 2019-12-13 21:13:14.307378
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '20c10335b553'
+down_revision = 'c11dc931b903'
+
+from alembic import op
+import sqlalchemy as sa  # NOQA
+
+
+def upgrade():
+    op.add_column(
+        'project',
+        sa.Column(
+            'livestream_urls',
+            sa.ARRAY(sa.UnicodeText(), dimensions=1),
+            server_default='{}',
```

Comment:
Is this the correct server default?

Reply:
Your model doesn't have a `server_default`, so the migration should not have one either. If you want to insert default values when adding the column, then alter it immediately after to remove the server default.